README.md

```diff
@@ -1,4 +1,201 @@
+| skip_generate | skip running all-contributors generate | false | false |
+
+If you aren't familiar with all-contributors, you'll need to add some
+[commenting in your repository README](https://allcontributors.org/docs/en/cli/usage)
+so the client knows where to write.
+
+## Docker Usage
+
+If you don't want to use the GitHub Action but want to interact with the tool
+in a container, you can do that too! 
+
+### 1. Prepare the Container
+
+First, build the container from the included
+Dockerfile:
+
+```bash
+docker build -t allcontributors .
+```
+
+Then shell inside, ensuring that the entrypoint is changed to bash. You'll also
+want to bind your repository to somewhere in the container (not `/code`).
+
+```bash
+docker run -it --entrypoint bash -v $PWD/:/data allcontributors
+```
+
+In the container, a2z is installed as a python "binary" on the path:
+
+```bash
+# which a2z
+/usr/local/bin/a2z
+```
+
+The all contributors client is also on the path:
+
+```bash
+$ which cli.js
+/code/node_modules/all-contributors-cli/dist/cli.js
+```
+
+### 2. Generate .all-contributorsrc
+
+Let's now change directory to where we bound our repository as a volume.
+
+```bash
+cd /data
+```
+
+If you don't have an `.all-contributorsrc` in your present working directory (the
+root of the repository) you can use [a2z.py](a2z.py) to generate an empty one. You
+can either provide the repository name directly:
+
+```bash
+$ a2z init singularityhub/sregistry
+INFO:a2z:Generating .all-contributorsrc
+```
+
+or export it to the environment as `GITHUB_REPOSITORY` (this is how it would be discovered
+during a GitHub workflow run).
+
+```bash
+export GITHUB_REPOSITORY=singularityhub/sregistry
+$ a2z init
+INFO:a2z:Generating .all-contributorsrc
+```
+In both cases, you'll generate an empty file:
+
+```bash
+$ cat .all-contributorsrc 
+{
+    "projectName": "sregistry",
+    "projectOwner": "singularityhub",
+    "repoType": "github",
+    "repoHost": "https://github.com",
+    "files": [
+        "README.md"
+    ],
+    "imageSize": 100,
+    "commit": true,
+    "commitConvention": "none",
+    "contributors": [],
+    "contributorsPerLine": 7
+}
+```
+
+### 2. Generate .zenodo.json
+
+If you want to generate a fresh Zenodo.json, you can do that as follows:
+
+```bash
+$ a2z init --zenodo 10.5281/zenodo.1012531 singularityhub/sregistry
+```
+
+While the GitHub repository is sometimes found as a related identifier,
+this isn't always the case, so you should again provide it on the command
+line or exported to `GITHUB_REPOSITORY`. By default, we parse contributors
+from the GitHub API, and include the creators already defined in Zenodo.
+You will need this `.zenodo.json` file to exist in order to update it.
+
+### 3. Update contributors
+
+We next want to use the GitHub api to discover contributors to the repository,
+for either both the `.all-contributorsrc` and the `.zenodo.json`, or just one of the two.
```

Comment:
Might not be needed if those are collected... but indeed could be of help to add contributors retroactively in existing repos!  But IMHO it should probably be an interactive interface (may be if `-i` cmdline option is given?) which would ask
- either to add that contributor to contributors (typically -- YES for all, but in some cases like in ohbm2020-posters we decided to not add people who merely provided urls to their posters)
- for what "allcontributors" categories (that is where their CLI is suboptimal btw since IIRC it doesn't even list choices).

Reply:
I can definitely see an interactive version.